app/Models/TravelAssignment.php

```diff
@@ -70,6 +74,43 @@ class TravelAssignment extends Model
         'documents_received' => 'boolean',
     ];
 
+    /**
+     * The attributes that should be searchable in Meilisearch.
+     *
+     * @var array<string>
+     */
+    public $searchable_attributes = [
+        'user_first_name',
+        'user_preferred_name',
+        'user_last_name',
+        'user_uid',
+        'user_gt_email',
+        'user_personal_email',
+        'user_gmail_address',
+        'user_clickup_email',
+        'user_autodesk_email',
+        'user_github_username',
+        'travel_name',
+        'travel_destination',
+        'travel_departure_date',
+        'travel_return_date',
+        'payable_type',
+    ];
+
+    /**
+     * The rules to use for ranking results in Meilisearch.
+     *
+     * @var array<string>
+     */
+    public $ranking_rules = [
+        'desc(user_revenue_total)',
+        'desc(user_attendance_count)',
+        'desc(user_signatures_count)',
+        'desc(user_recruiting_visits_count)',
+        'desc(user_gtid)',
```

Comment:
Not sure but sorting by user last name might be more intuitive?  I think the GTID sort would create the effect of a seemingly random ordering by name

Reply:
Custom ranking rules need to be on numeric fields.